ci/it/testcases/test_ingest.go

```diff
@@ -645,3 +654,423 @@ func (a *IngestTestcase) testFieldEncodingsMappingsBug(ctx context.Context, t *t
+					} else {
+						addError(fmt.Sprintf("field %s not found in response", fieldName))
+						r.querySuccess = false
+					}
+				} else {
+
+					var fieldValueAsString string
+					switch v := fieldValue.(type) {
+
+					case string:
+						fieldValueAsString = strconv.Quote(v)
+					case float64:
+						fieldValueAsString = strconv.FormatFloat(v, 'f', -1, 64)
+
+					default:
+						data, err := json.Marshal(v)
+						if err != nil {
+
+						}
+						fieldValueAsString = string(data)
+					}
+
+					if fieldValueAsString != typ.ingestValue {
+						r.querySuccess = false
+						addError(fmt.Sprintf("field %s has unexpected value %v", fieldName, fieldValueAsString))
+					}
+
+				}
+			}
+
+			columns, err := a.FetchClickHouseColumns(ctx, indexName)
+			columName := strings.ReplaceAll(fieldName, "-", "_")
+			if err != nil {
+				t.Fatalf("failed to fetch 'quesma_common_table' columns: %v", err)
+			} else {
+				if dbType, ok := columns[columName]; ok {
+					r.dbStorage = "single column: " + dbType
+				} else {
+					r.dbStorage = "n/a"
+					prefix := columName + "_"
+
+					var cols []string
+					for k, _ := range columns {
+						if strings.HasPrefix(k, prefix) {
+							cols = append(cols, k)
+						}
+					}
+					if len(cols) > 0 {
+						r.dbStorage = "columns: " + strings.Join(cols, ", ")
+					}
+				}
+			}
+
+			r.currentSupport = len(r.errors) == 0
+
+			switch {
+
+			case !r.claimedSupport && r.currentSupport:
+				t.Log("Type not supported but works. What a surprise!")
+
+			case r.claimedSupport && r.currentSupport:
+				t.Log("Type supported and works. All good.")
+
+			case !r.claimedSupport && !r.currentSupport:
+				t.Skip("Type not supported and it doesn't work. Not great. Not terrible.")
+
+			case r.claimedSupport && !r.currentSupport:
+				t.Errorf("Type '%s' should be supported but is not: %v", r.name, r.errors)
+			}
+
+		})
+	}
+
+	sort.Slice(results, func(i, j int) bool {
+		return results[i].name < results[j].name
+	})
+
+	fmt.Println("")
+	// Create a new tabwriter
+	w := tabwriter.NewWriter(os.Stdout, 0, 0, 2, ' ', 0)
+
+	// Print table header
+	fmt.Fprintf(w, "Name\tSupport\tCurrent Support\tPut Mapping\tIngest\tQuery\tStored as\t\n")
+	fmt.Fprintf(w, "----\t-------\t---------------\t-----------\t------\t-----\t---------\t\n")
+
+	// Print rows
+	for _, res := range results {
+		fmt.Fprintf(w, "%s\t%v\t%v\t%v\t%v\t%v\t%v\t\n",
+			res.name, res.claimedSupport, res.currentSupport, res.putMappingSuccess, res.ingestSuccess, res.querySuccess, res.dbStorage)
+	}
+
+	// Flush the writer to output
+	w.Flush()
+
+	fmt.Println("")
+
+	var failedTypes []string
+
+	for _, r := range results {
+
+		if r.claimedSupport && !r.currentSupport {
+			failedTypes = append(failedTypes, r.name)
+		}
+		if len(r.errors) > 0 {
+			fmt.Println("Type: ", r.name)
+			fmt.Println("Errors: ", strings.Join(r.errors, ", "))
+		}
+	}
+}
```

Comment:
We don't do anything with `failedTypes`. Maybe forgot to print?

Reply:
This is leftover. Test prints a summary table instead.